src/redesign/_style.scss

```diff
@@ -532,15 +532,17 @@ select.form-control {
 }
 
 .username-suggestion {
-  padding: 2px 0.5rem;
+  padding: 1px 0.5rem;
```

Comment:
nit: Might need to update padding-top to 2px as Figma shows padding `padding: 2px 8px;`.

Reply:
@shafqatfarhan it increases the height to 30px but Figma suggests 28px.